services/src/main/java/org/exoplatform/news/storage/jcr/JcrNewsStorage.java

```diff
@@ -250,7 +250,7 @@ public News createNews(News news) throws Exception {
     }
     publicationService.enrollNodeInLifecycle(newsDraftNode, lifecycleName);
     publicationService.changeState(newsDraftNode, PublicationDefaultStates.DRAFT, new HashMap<>());
-    newsDraftNode.setProperty("exo:body", imageProcessor.processImages(news.getBody(), newsDraftNode.getUUID(), "images"));
+    newsDraftNode.setProperty("exo:body", news.getBody());
```

Comment:
We need to make sure to save Draft with persisted Image to not loose it when we restart the server, since the Uploaded resources will be added into `temp` folder of Tomcat